CMakeLists.txt

```diff
@@ -29,28 +31,44 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR}/log4tango/include)
 #source code
 add_subdirectory("log4tango")
 add_subdirectory("cppapi")
-add_subdirectory("cpp_test_suite")
+#add_subdirectory("cpp_test_suite")
```

Comment:
We can tolerate the tests to be deactivated on Windows for the moment, but they should be compiled on Linux and they should be executed on Travis CI

Reply:
Done added a if not win32